pkg/nsx/services/staticroute/staticroute.go

```diff
@@ -137,11 +137,11 @@ func (service *StaticRouteService) DeleteStaticRoute(namespace string, uid strin
 	return service.DeleteStaticRouteByPath(path[2], path[4], *vpc[0].Id, uid)
 }
 
-func (service *StaticRouteService) ListStaticRoute() []model.StaticRoutes {
+func (service *StaticRouteService) ListStaticRoute() []*model.StaticRoutes {
```

Comment:
For store, we use ListIndexFuncValues instead of list() function from strore, seemly this way is better. Of course, using list() also might be ok.

Reply:
ListIndexFuncValues will return key or id? for staticroute, the delete needs model.StaticRoutes.Path/Id, so here use List()